lib/archive-utils.rb

```diff
@@ -20,5 +15,4 @@ module Archive
 require 'archive/fixity'
 require 'archive/operating_system'
 require 'archive/tarfile'
-include Archive
```

Comment:
And, as far as I can tell, `sdr-replication` and `sdr-preservation-core` are the only consumers of this gem, and are trivially updated in response to this change.